Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/CompoundBocListValidationFailureHandlerTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Moq;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+using Remotion.ObjectBinding.Web.UI.Controls.Validation;
+
+namespace Remotion.ObjectBinding.Web.UnitTests.UI.Controls.Validation
+{
+  [TestFixture]
+  public class CompoundBocListValidationFailureHandlerTest
+  {
+    [Test]
+    public void HandleValidationFailures_CallsContainedHandlers ()
```

Comment:
Please add VerifiableSequence to the test. The order is relevant during execution.
see CompoundExtensibleEnumGlobalizationServiceTest